run_service.sh

```diff
@@ -562,7 +660,13 @@ fi
 # activate service
 if [ "$(get_on_chain_service_state "$service_id")" == "PRE_REGISTRATION" ]; then
     echo "[Service owner] Activating registration for on-chain service $service_id..."
-    output=$(poetry run autonomy service --use-custom-chain activate --key "../$operator_pkey_path" "$service_id")
+    export cmd="poetry run autonomy service --use-custom-chain activate --key "../$operator_pkey_path" "$service_id""
+    if [ "${use_staking}" = true ]; then
+        minimum_olas_balance=1000000000000000000
```

Comment:
For future version of this script, we should get this from the staking contract. So I think the script should have an env var with the staking contract and we pull this info dynamically from there.